source/val/validate_decorations.cpp

```diff
@@ -1570,6 +1586,9 @@ spv_result_t CheckDecorationsFromDecoration(ValidationState_t& vstate) {
         case SpvDecorationNoUnsignedWrap:
           PASS_OR_BAIL(CheckIntegerWrapDecoration(vstate, *inst, decoration));
           break;
+        case SpvDecorationBlock:
```

Comment:
Based on the error message, this should also include SpvDecorationBufferBlock it seems.

Reply:
You are correct. Fixed.